services/api-server/Makefile

```diff
@@ -138,17 +164,21 @@ null  :=
 space := $(null) #
 comma := ,
 
+CONTAINER_TEMPL_DIR := /tmp/openapi_templates
+export DOCKER_MOUNT := $(REPO_BASE_DIR)/services/api-server/openapi/python_templates:$(CONTAINER_TEMPL_DIR)
```

Comment:
It is not very clear from the context that this env var is used in `$(SCRIPTS_DIR)/openapi-generator-cli.bash`

Some options:

- adding it just before `$(SCRIPTS_DIR)/openapi-generator-cli.bash` with a clear name
```
OPENAPI_GENERATOR_CLI_MOUNT=$(REPO_BASE_DIR)/services/api-server/openapi/python_templates:/tmp/template-dir 
$(SCRIPTS_DIR)/openapi-generator-cli.bash generate ...
```
- Always mounting `$(PWD):/workdir`  (see e.g. https://github.com/ITISFoundation/osparc-simcore/blob/cb8779e62941f8c442a070a0759fef88b7ec674a/scripts/mypy.bash)